toDoList.html

```diff
@@ -0,0 +1,55 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <title>To-Do List</title>
+        <link rel='stylesheet' href='https://use.fontawesome.com/releases/v5.0.6/css/all.css'>
+    </head>
+    <body>
+
+        <div id='header'>
+            <img class='logo' id='logo' src='EventMobiLogo.svg' alt='EventMobi'>
+        </div>
+
+        <div id='content'>
+
+            <div>
+                Event Planner To-Do List
+            </div>
+
+            <div class='to-do-item'>
+                <button type='button' class='btn-del-single btn-danger btn-base task-option'><i class='fa fa-trash-o' aria-hidden='true'></i> Delete</button>
```

Comment:
break these tags to be one line each for readability 
```
<button>
   <i class...></i> Delete
...
```
same applies everywhere else in this file